robyn/__init__.py

```diff
@@ -99,7 +104,7 @@ def after_request(self, endpoint: Optional[str] = None) -> Callable[..., None]:
         """
         You can use the @app.after_request decorator to call a method after routing to the specified endpoint
 
-        :param endpoint str|None: endpoint to server the route. If None, the middleware will be applied to all the routes.
```

Comment:
Is this required?

Reply:
I didn't write those comments. They are present in the original Robyn __init__.py too.